### ISSUE TITLE
placeholder text

### ISSUE THREAD

**Issue by mistohise** [freenode/gms#91]
Placeholder text such as "There isn't much here just now." needs to be replaced with informative context-relevant text. Such as "Please log in" or "Please use the buttons to the left to choose an option", or whatever.

**Comment by mniip** [comment]
Re: https://github.com/freenode/gms/commit/2040659822723b65a30bf650193c650d9a9e810d
I don't think every other user knows what 'Atheme' is. Maybe change to 'NickServ username and password'?

**Comment by mistohise** [comment]
Agreed. Something like "Please log in with your freenode nickserv username and password", or another phrasing of that that's less clunky.

**Comment by mistohise** [comment]
Additionally, there is also placeholder text on the page known as "GMS home".

**Comment by christeld** [comment]
I'll sort out the wording of the various bits over the next while :)

**Comment by errietta** [comment]
@christeld we want our copy
